docs/apm.asciidoc

```diff
@@ -172,13 +172,13 @@ spec:
 
 By default the operator manages a private CA and generates a self-signed certificate used to secure the communication between APM agents and the server.
 
-This behavior and the relevant configuration is identical to what is done for Elasticsearch and Kibana. You can refer to link:k8s-accessing-elastic-services.html#k8s-setting-up-your-own-certificate[this documentation] to understand how to use your own certificate to configure the TLS endpoint of the APM server.
+This behavior and the relevant configuration is identical to what is done for Elasticsearch and Kibana. You can refer to link:k8s-accessing-elastic-services.html#k8s-setting-up-your-own-certificate[this documentation] to understand how to use your own certificate to configure the TLS endpoint of the APM Server.
```

Comment:
In a default setup APM Agents connect to the APM Server, rather than users sending custom requests. Since agents need configuration to work with custom certificates, I suggest to also add a note pointing this out, and linking to the [agents overview doc](https://www.elastic.co/guide/en/apm/agent/index.html). Users can take it from there and dive into whichever language they use and find specific documentation e.g. [#config-server-cert](https://www.elastic.co/guide/en/apm/agent/python/current/configuration.html#config-server-cert)